gradle.properties

```diff
@@ -15,3 +15,7 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+
+SAMPLE_VERSION_NAME=0.0.10
+
+KIN_ECOSYSTEM_SDK_VERSION_NAME=0.0.2
```

Comment:
when we will increment it? when creating a release TAG?

Reply:
Sample app will be incremented every delivery to QA.
SDK - will be incremented every new release TAG yes.